pkgs/servers/web-apps/pict-rs/default.nix

```diff
@@ -13,21 +13,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "pict-rs";
-  version = "0.3.3";
+  version = "0.4.0";
 
   src = fetchFromGitea {
     domain = "git.asonix.dog";
     owner = "asonix";
     repo = pname;
     rev = "v${version}";
-    sha256 = "mEZBFDR+/aMRFw54Yq+f1gyEz8H+5IggNCpzv3UdDFg=";
+    sha256 = "sha256-1WNd7Ei21g01S5ko6y+uyhHP+xlGtnrwU8MLMxnW3P8=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
```

Comment:
You can actually delete the cargo.lock again. Yes, importing from it saves some binary cache storage but it comes at the cost of evaluation memory and nixpkgs tarball size.

Reply:
Done.